src/store/index.js

```diff
@@ -20,8 +21,6 @@ const initialState = {
   account: {},
   mnemonic: null,
   activeAccount: 0,
-  names: [],
```

Comment:
Don't we need to create a migration to don't break the state of existing users?

Reply:
Only migration for default name will be needed i think. Names will be fetched as soon as application is started